account/src/main/java/com/caldevsupplychain/account/service/AccountService.java

```diff
@@ -0,0 +1,40 @@
+package com.caldevsupplychain.account.service;
+
+
+
+import com.caldevsupplychain.account.model.Company;
+import com.caldevsupplychain.account.model.User;
+import com.caldevsupplychain.common.ws.account.UserWS;
+
+import javax.swing.text.html.Option;
+import java.util.Optional;
+
+public interface AccountService {
+
+    boolean userExist(String emailAddress);
+
+    User createUser(UserWS userWS);
```

Comment:
Return `UserBean`,  Change `UserWS`  to `UserBean`. Convert ws to bean before calling this method.

Reply:
Fixed